catalogue/webapp/hooks/useIIIFManifestData.ts

```diff
@@ -40,52 +50,88 @@ function parseManifest(manifest: IIIFManifest): IIIFManifestData {
     'mediaDownload'
   );
   const firstChildManifestLocation = getFirstChildManifestLocation(manifest);
+
   return {
     imageCount,
     childManifestsCount,
-    audioItems,
+    audio,
     video,
     iiifCredit,
     iiifPresentationDownloadOptions,
     iiifDownloadEnabled,
     firstChildManifestLocation,
+    manifestV3,
   };
 }
 
 const startedFetchingIds = new Set();
```

Comment:
What does this do?

Reply:
It makes the whole manifest available from the hook, as well as the various manipulated parts (needed for a [couple of functions in `WorkDetails`](https://github.com/wellcomecollection/wellcomecollection.org/blob/multiple-audio-v3/catalogue/webapp/components/WorkDetails/WorkDetails.tsx#L139-L148)). I suppose I could return just the new bit that's needed (services), but don't think there's harm in returning the whole manifest?

Reply:
Sorry I think I commented on the wrong bit - what does the `Set` do?

Reply:
Not sure – possibly a way to collect de-duped ids, or something? I don't think I touched it in this PR